Code/Engine/Foundation/Time/Time.h

```diff
@@ -15,33 +15,40 @@ struct EZ_FOUNDATION_DLL ezTime
   static ezTime Now(); // [tested]
 
   /// \brief Creates an instance of ezTime that was initialized from nanoseconds.
-  EZ_ALWAYS_INLINE constexpr static ezTime Nanoseconds(double fNanoseconds) { return ezTime(fNanoseconds * 0.000000001); }
+  [[nodiscard]] EZ_ALWAYS_INLINE constexpr static ezTime MakeNanoseconds(double fNanoseconds) { return ezTime(fNanoseconds * 0.000000001); }
```

Comment:
Shouldn't that be MakeFromNanoseconds etc. to follow the naming scheme of the other classes?

Reply:
Theoretically yes, but that makes it even longer. I don't know, how do you feel about it ?

Reply:
I would prefer consistency in this case

Reply:
Done